.azure/templates/jobs/build_docs.yaml

```diff
@@ -0,0 +1,41 @@
+jobs:
+  - job: 'build_docs'
+    displayName: 'Build Docs'
+    # Strategy for the job
+    # Set timeout for jobs
+    timeoutInMinutes: 60
+    # Base system
+    pool:
+      vmImage: 'Ubuntu-20.04'
+    # Variables
+    variables:
+      MVN_CACHE_FOLDER: $(HOME)/.m2/repository
+      MVN_ARGS: '-e -V -B'
+    # Pipeline steps
+    steps:
+      - task: Cache@2
+        inputs:
+          key: 'mvn-m2-cache | $(System.JobName)'
+          path: "$(MVN_CACHE_FOLDER)"
+        displayName: Maven cache
+      - template: '../steps/setup_java.yaml'
+        parameters:
+          JDK_PATH: '/usr/lib/jvm/java-11-openjdk-amd64'
+          JDK_VERSION: '11'
+      - template: '../steps/setup_asciidoc.yaml'
+#            - bash: "make docu_check"
+#              displayName: "Docu checks"
+#              env:
+#                MVN_ARGS: "-e -V -B"
+#              condition: and(succeeded(), eq(variables['main_build'], 'true'))
```

Comment:
this commented part has to be here for any future reference or could be deleted?